_posts/en/newsletters/2025-11-28-newsletter.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Bitcoin Optech Newsletter #382'
+permalink: /en/newsletters/2025/11/28/
+name: 2025-11-28-newsletter
+slug: 2025-11-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Stats on compact block reconstructions updates:** 0xB10C posted to
+  [Delving Bitcoin][0xb10c delving] about his stats on compact block
+  reconstruction. This is an update to a previous [optech
+  post][compact block optech], since then 0xB10C has provided three updates
+  on compact block reconstruction. He has also tracking the average size of
+  requested transactions in kB, because of the [previous
+  post][david delving post] by Gumberg. The new chart is similar looking as
+  the share of compact block reconstructions without request transactions
+  chart and can be found directly under them in each post.
+
+  0xB10C updated one of his nodes named bob to the master branch and found
+  that because there were no existing sub-1 sat/vbyte transactions there was
+  no improvment for the first few days, afterwards block reconstruction rates
```

Comment:
```suggestion
  that because there were no existing sub-1 sat/vbyte transactions, there was
  no improvement for the first few days; afterwards block reconstruction rates
```